sphinxcontrib/confluencebuilder/storage/translator.py

```diff
@@ -2855,6 +2855,37 @@ def visit_vimeo(self, node):
     def visit_youtube(self, node):
         self._visit_video(node, 'https://www.youtube.com/watch?v=' + node['id'])
 
+    # ---------------------------------------------------
+    # sphinx -- extension (third party) -- sphinx-video
+    # ---------------------------------------------------
+
+    def visit_video_node(self, node):
+        autoplay = node.get("autoplay")
```

Comment:
(optional) This extension typically uses single quotes over double quotes when possible.